models/review-model.js

```diff
@@ -27,7 +27,25 @@ exports.updateVoteCount = (review_id, inc_votes) => {
 exports.selectAllReview = () => {
     return db.query(`SELECT reviews.review_id, reviews.title, reviews.designer,owner, reviews.review_img_url, reviews.category, reviews.created_at, reviews.votes, COUNT (comments.review_id) ::INT AS comment_count FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id WHERE reviews.review_id = reviews.review_id GROUP BY reviews.review_id  ORDER BY created_at DESC`)
     .then(({ rows }) => {
-        console.log(rows);
         return rows;
     })
-}
\ No newline at end of file
+}
+
+exports.selectReviewComment = async (review_id) => {
+    const result = await db.query(`SELECT comments.comment_id, comments.created_at, comments.body, comments.votes, comments.review_id, comments.author FROM comments WHERE comments.review_id = $1 `, [review_id])
+        
+        
+    const result2 = await db.query(`SELECT reviews.review_id FROM reviews GROUP BY reviews.review_id`) 
+        
+         
+    if (review_id > result2.rowCount){
+        return Promise.reject({ status: 404, msg: "page not found"})
```

Comment:
I dont know what this is checking for? example: review_id === 2 but rowCount === 1?

It would be better to just check the that rowCount is not 0, or the length of result2.rows to find out if it exists

Reply:
Ok will refactor now :)